physionet-django/events/templates/events/event_home.html

```diff
@@ -114,7 +117,46 @@ <h5 class="modal-title">Participants</h5>
             </div>
           </div>
         </div>
-    {% endfor %}
+        <div class="modal fade" id="pending-applications-modal-{{ event.id }}" tabindex="-1" role="dialog" aria-labelledby="pending-applications-modal" aria-hidden="true">
+          <div class="modal-dialog modal-lg"  role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title">Pending Applications</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+                {% include 'events/event_pending_applications.html' %}
+              </div>
+            </div>
+          </div>
+          <div class="modal fade" id="rejected-applications-modal-{{ event.id }}" tabindex="-1" role="dialog" aria-labelledby="rejected-applications-modal" aria-hidden="true">
+            <div class="modal-dialog modal-lg"  role="document">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h5 class="modal-title">Rejected Applications</h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                  </button>
+                </div>
+                  {% include 'events/event_rejected_applications.html' %}
+                </div>
+              </div>
+            </div>
+            <div class="modal fade" id="withdrawn-applications-modal-{{ event.id }}" tabindex="-1" role="dialog" aria-labelledby="withdrawn-applications-modal" aria-hidden="true">
+              <div class="modal-dialog modal-lg"  role="document">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <h5 class="modal-title">Withdrawn Application</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                  </div>
+                    {% include 'events/event_withdrawn_applications.html' %}
+                  </div>
+                </div>
+              </div>
```

Comment:
Similarly to the modals in the admin panel, this could be dynamically rendered using a list of dictionaries. (the buttons should also reference the IDs. The buttons could be dynamically rendered as well to maintain the id<->id consistency.